cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -363,8 +363,8 @@ const char* gdv_fn_aes_decrypt(int64_t context, const char* data, int32_t data_l
     *out_len = gandiva::aes_decrypt(data, data_len, key_data,
                                     reinterpret_cast<unsigned char*>(ret));
   } catch (const std::runtime_error& e) {
-    gdv_fn_context_set_error_msg(context, e.what());
-    return nullptr;
+    std::string err_msg = "Error occurred while decrypting ciphertext.";
```

Comment:
Please include the original error message as much as possible. For e.g. something like
"Error occurred while decrypting ciphertext: %s", e.what()

Reply:
Can you also search for this pattern in other functions and ensure that we fix those instances also?

Other functions that catch runtime_error might have similar issues